protocols/indexer/contracts/Indexer.sol

```diff
@@ -54,6 +56,15 @@ contract Indexer is IIndexer, Ownable {
     locatorWhitelist = _locatorWhitelist;
   }
 
+  /**
+    * @notice Modifier to prevent function call unless the contract is not paused
+    *
+    */
+  modifier whenNotPaused() {
```

Comment:
Suggest `notPaused` to be consistent with `onlyOwner`

Reply:
This meant I have to rename the `bool paused` to `bool contractPaused` to keep the other modifier consistent.